src/css/components/_progress.scss

```diff
@@ -86,12 +86,12 @@
 .video-js .vjs-play-progress:after {
   display: none;
   position: absolute;
-  top: -2.4em;
+  top: -3.5em;
   right: -1.5em;
   font-size: 0.9em;
   color: #000;
   content: attr(data-current-time);
-  padding: 0.2em 0.5em;
+  padding: 6px 8px 8px 8px;
```

Comment:
What browser? This is what I'm seeing in Chrome:

![screen shot 2016-02-01 at 2 53 58 pm](https://cloud.githubusercontent.com/assets/211047/12729227/b0921cca-c8f3-11e5-9a27-eb517f8082b3.png)